deploy/src/main/deploy/helm/values.yaml

```diff
@@ -3,16 +3,28 @@
 # Declare variables to be passed into your templates.
 
 # amqpMessagingNetworkDeployExample indicates whether the example AMQP Messaging Network
-# consisting of a single Dispatch Router and Artemis broker should be
+# consisting of a single Dispatch Router and Artemis broker (or Azure ServiceBus) should be
```

Comment:
I don't want to play Azure role by any means, but based on your discussions, maybe we can make this general external AMQP broker option? So, you can either use internally deployed Artemis broker, or configure parameters to the external one?

Reply:
I like the idea, so _amqpMessagingNetworkDeployExample_  would refer to the router only and then there is an internal and external option on the broker?.

I will create a proposal.

Reply:
@dejanb , @sophokles73 thoughts?

```yaml
# amqpMessagingNetworkExample indicates whether the example AMQP Messaging Network
# consisting of a single Dispatch Router and Broker should be
# deployed and used. By default an internal Broker is deployed.
# As alternative an external Broker can be configured as well.
amqpMessagingNetworkExample:
  enabled: true
  # dispatchRouter.imageName contains the name (including tag)
  # of the container image to use for the example AQMP Messaging Network Router.
  dispatchRouter:
    imageName: ${dispatch-router.image.name}
    svc:
      annotations: {}
      loadBalancerIP:
  # internalBroker.imageName contains the name (including tag) of the container
  # image to use for the example AMQP Messaging Network Broker
  internalBroker:
    enabled: true
    imageName: ${artemis.image.name}
  # alternative external Broker. A qrouterd configuration file of 'type' has to be available.
  externalBroker:
    enabled: false
    type: servicebus
    saslUsername:
    saslPassword:
    host:
```

Based on the comment above I would then introduce a broker config "qrouterd-with-servicebus.yaml"

Reply:
@kaizimmerm This looks good to me. Thanks

Reply:
@dejanb after testing the varaiants I managed to get it even simpler. Plus the the _amqpMessagingNetworkExample.broker.type_ value drives the decision over the router's config file.

```yaml
# amqpMessagingNetworkExample indicates whether the example AMQP Messaging Network
# consisting of a single Dispatch Router and Broker should be
# deployed and used. By default an internal Broker is deployed.
# As alternative an external Broker can be configured as well.
amqpMessagingNetworkExample:
  enabled: true
  # dispatchRouter.imageName contains the name (including tag)
  # of the container image to use for the example AQMP Messaging Network Router.
  dispatchRouter:
    imageName: ${dispatch-router.image.name}
    svc:
      annotations: {}
      loadBalancerIP:
  # AMQP Messaging Network Broker configuration.
  broker:
    type: artemis
    artemis:
    # imageName contains the name (including tag) of the container
    # image to use for the example AMQP Messaging Network Broker
      imageName: ${artemis.image.name}
   # Azure Service Bus as alternative to self-hosted Artemis ActiveMQ for event persistence
    servicebus:
      saslUsername:
      saslPassword:
      host:
```

Reply:
@kaizimmerm That indeed looks clean 👍